lib/rspec/core/configuration.rb

```diff
@@ -199,6 +199,10 @@ def only_failures_but_not_configured?
       # The exit code to return if there are any failures (default: 1).
       add_setting :failure_exit_code
 
+      # @macro add_setting
+      # Whether or not to fail when there are no RSpec examples (default: false).
+      add_setting :fail_no_examples
```

Comment:
Currently the build is failing due to a drop in test coverage, I believe this needs a test covering it to fix that :)